src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/Given_ProcessStartInfoType.cs

```diff
@@ -0,0 +1,223 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace System.Diagnostics.ProcessTests
+{
+    public class Given_ProcessStartInfoType : ProcessTestBase
+    {
+        [Fact]
+        public void Then_EnvironmentShouldBe()
+        {
+            Assert.NotEqual(0, new Process().StartInfo.Environment.Count);
+
+            ProcessStartInfo psi = new ProcessStartInfo();
+
+            // Creating a detached ProcessStartInfo will pre-populate the environment
+            // with current environmental variables.
+
+            var Environment2 = psi.Environment;
+
+            Assert.NotEqual(Environment2.Count, 0);
+
+            int CountItems = Environment2.Count;
+
+            Environment2.Add("NewKey", "NewValue");
+            Environment2.Add("NewKey2", "NewValue2");
+
+            Assert.Equal(CountItems + 2, Environment2.Count);
+            Environment2.Remove("NewKey");
+            Assert.Equal(CountItems + 1, Environment2.Count);
+
+            //Exception not thrown with invalid key
+            Assert.Throws<ArgumentException>(() => { Environment2.Add("NewKey2", "NewValue2"); });
+
+            //Clear
+            Environment2.Clear();
+            Assert.Equal(0, Environment2.Count);
+
+            //ContainsKey 
+            Environment2.Add("NewKey", "NewValue");
+            Environment2.Add("NewKey2", "NewValue2");
+            Assert.True(Environment2.ContainsKey("NewKey"));
+            if (global::Interop.IsWindows)
+            {
+                Assert.True(Environment2.ContainsKey("newkey"));
```

Comment:
Should this instead be:
```C#
Assert.Equal(global::Interop.IsWindows, Environment2.ContainsKey("newkey"));
```
so that on Windows we're validiting it's found and on Unix we're validating it's not?